h2o-hadoop/h2o-cdh6.0/CDH60.Makefile

```diff
@@ -0,0 +1,100 @@
+CDHV=6.0
```

Comment:
does this file belong in this directory? no other hadoop distributions have it

cc: @michal-raska

Reply:
No, I think this is a "helper", will remove.